### ISSUE TITLE
Focus Visible Failures

### ISSUE THREAD

**Issue by Detroit-the-Dev** [tsdataclinic/scout#306]
**Describe the bug**
Keyboard users who use the tab key to navigate through page aren't able to see which dataset they are focused on on the page. The elements ARE focused but not visibly; updated to include outline for visual tracking.

**To Reproduce**
1. Go to: https://scout.tsdataclinic.com/explore/NYC
2. Advance through screen using Tab key to 1st dataset link.
3. Unable to see where I was on the page as there's no outline to indicate focus.
4. Added outline to dataset link to allow for visual tracking of focused element.

**Expected behavior**
User should be able to see where their tab focus is when navigating through the page.

**Screenshots**

<img width="1440" alt="Before Dataset Link Outline" src="https://user-images.githubusercontent.com/70827740/182431519-594830c2-29df-446d-a266-c360d7367f17.png">

![Dataset Link With Outline](https://user-images.githubusercontent.com/70827740/182431561-7659f136-0a3a-4d59-bdad-3f8eb9178b9d.jpeg)

**Additional context**
